src/app/pages/datasets/components/space-management-chart/space-management-chart.component.html

```diff
@@ -1,13 +1,20 @@
-<div class="chart-wrapper" *ngIf="!isLoading; else chartPlaceholder">
-  <canvas
-    #canvas
-    baseChart
-    chartType="doughnut"
-    [datasets]="chartData"
-    [options]="chartOptions"
-    [width]="180"
-    [height]="180"
-  ></canvas>
+<div>
+  <div class="chart-wrapper" *ngIf="!isLoading; else chartPlaceholder">
+    <canvas
+      #canvas
+      baseChart
+      chartType="doughnut"
+      [datasets]="chartData"
+      [options]="chartOptions"
+      [width]="180"
+      [height]="180"
+    ></canvas>
+  </div>
+  <div class="provision-wrapper" *ngIf="!isZvol">
+    <span class="provision-label">{{ 'Provisioning Type' | translate }}:</span>
```

Comment:
It is part of the parent component. A few days ago, I left a [comment](https://ixsystems.atlassian.net/browse/NAS-117278?focusedCommentId=152140) under the ticket.

https://github.com/truenas/webui/blob/c542ac6a57e5c2748ffda7ccad464e58164958b5/src/app/pages/datasets/components/dataset-capacity-management-card/dataset-capacity-management-card.component.html#L24-L28